src/invoices/InvoiceDetails/constants.js

```diff
@@ -3,3 +3,7 @@ export const VENDOR_STATUS = {
   INACTIVE: 'Inactive',
   PENDING: 'Pending',
 };
+
+export const ORDER_STATUS = {
```

Comment:
You can use the constant from the `stripes-acq-components` lib https://github.com/folio-org/stripes-acq-components/blob/master/lib/constants/order.js#L4-L8